pyacq/core/stream.py

```diff
@@ -154,33 +262,55 @@ def __init__(self, **kargs):
         #send or cpy to buffer
         if self.params['transfermode'] == 'plaindata':
```

Comment:
Should probably break StreamSender and StreamReceiver up into subclasses (one class per transfer mode) rather than mix all code into monolithic classes?